--- rusty_paseto-0.6.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ rusty_paseto-0.6.1/Cargo.toml	2024-05-28T09:02:59.832142+00:00
@@ -50,11 +50,11 @@
 optional = false

 [dependencies.blake2]
-version = "^0.9.2"
+version = ">=0.9.2"
 optional = true

 [dependencies.chacha20]
-version = "0.8.1"
+version = ">=0.8.1"
 optional = true

 [dependencies.chacha20poly1305]
@@ -116,9 +116,6 @@
 [dependencies.zeroize]
 version = "1.4.3"
 features = ["zeroize_derive"]
-
-[dev-dependencies.actix-identity]
-version = "0.4.0"

 [dev-dependencies.actix-utils]
 version = "3.0.0"
